newIDE/app/src/InAppTutorial/InAppTutorialOrchestrator.js

```diff
@@ -1047,14 +1090,19 @@ const InAppTutorialOrchestrator = React.forwardRef<
     useInterval(watchSceneObjects, sceneObjectCountToWatch ? 1000 : null);
     useInterval(
       watchDomForNextStepTrigger,
-      currentStep && currentStep.isTriggerFlickering ? 500 : null
+      currentStep && currentStep.isTriggerFlickering ? 500 : 1000
```

Comment:
Why this change?

Reply:
Forgot to mention this, I had the issue where the MutationObserver was not triggered at all when switching scenes (no idea why) It seems to happen consistently.
I was wondering why we have this prop `isTriggerFlickering` ? It doesn't feel too costly to always check regularly the dom, in case the observer didn't catch the modification, as a fallback?

Reply:
Looking at the fling tutorial, it was needed when switching scenes as you mention, and also for the Add asset button in the asset store. I'm not sure if it's still needed indeed